contrib/make_apk

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 pushd lib
-VERSION=$(python -c "import version; print version.PACKAGE_VERSION")".0"
+VERSION=$(python -c "import version; \
+                     v = version.PACKAGE_VERSION.split('.'); \
+                     v = v + ['0'] * (3 - len(v)); \
+                     print '.'.join(v)")
```

Comment:
Wow, nice work! This is python 2 though, can you specify `python2` or specify `python3` and upgrade the syntax?

Reply:
Thanks. A new commit submitted using print function instead of print statement. It should work now in python2 as well as in python3.